labellab_mobile/lib/screen/project/detail/project_detail_screen.dart

```diff
@@ -560,15 +564,15 @@ class ProjectDetailScreen extends StatelessWidget {
             name: label.name,
             onCancel: () => Navigator.pop(context),
             onConfirm: () {
-              Provider.of<ProjectDetailBloc>(baseContext)
-                  .deleteLabel(label.labelID);
+              Provider.of<ProjectDetailBloc>(baseContext).deleteLabel(label.id);
               Navigator.of(context).pop(true);
             },
           );
         });
   }
 
-  void _showAddEditLabelModel(BuildContext baseContext, Label label) {
+  void _showAddEditLabelModel(BuildContext baseContext,
+      {String projectId, List<LabelLab.Image> images, Label label}) {
```

Comment:
@cazci Image argument is unused here. Is there a reason for this?

Reply:
@cazci Can you address this, please?